public/data/Yohn.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "Yohn aka John Brittain III",
+  "type": "personal",
+  "bio": "Self taught w3b enthusiast that's ready to break out with some new and exciting techniques, never seen before on the web, that seem to just 'work'.",
```

Comment:
Hmm, do you mean `Web3` when saying `w3b`, or what does that mean? 🤔

Reply:
Nope, I meant it with the w3b, where the '3' would be replacing the 'e', but in L337 verbiage, aka Leet, or elite. 

I mentioned w3b again in the milestones at the bottoms because I'm starting my own business - JW3B, which are my initials JWB III (or the 3rd), but since I could spell w3b with the 3 moved, I started going with it.